### PR TITLE
Feature/create recipe form styling

### PR DESCRIPTION
**Pull request by dlhauer**
- adds new icons for `+` and `x`
- standardizes borders and margins
- initializes form with 3 empty ingredients and 3 empty instructions
- adds highlighting to ingredient fields, except for `unit`, since Picker components don't include `onFocus` prop

**Note:** This PR does not complete styling for Create Recipe form. Two tasks remain, and should probably be handled on separate branches:

1. converting `Course Type` tags to dropdown/checkbox component
2. adding highlighting to all input fields, including ingredient `unit` Picker, if possible.

### CODE REVIEW

File: components/CreateRecipeForm.js [hunk 1]
```diff
@@ -32,14 +32,19 @@ import { Analytics, Event } from "expo-analytics";
...
     const initialFormState = {
         img: "",
         title: "",
         prep_time: "",
         cook_time: "",
         tags: [],
-        ingredients: [{ name: "", quantity: "", units: "" }],
-        instructions: [""],
+        ingredients: new Array(3).fill(emptyIngredient),
+        instructions: new Array(3).fill(""),
```
Comment (kmcknight1) [inline components/CreateRecipeForm.js]: Killer 🥇

File: components/RecipeName.js [hunk 2]
```diff
@@ -7,13 +7,13 @@ const RecipeName = ({ recipe, setRecipe, missing }) => {
     return (
         <>
             <View style={styles.heading}>
-                <Text>Title</Text>
+                <Text>Recipe Name</Text>
```
Comment (kmcknight1) [inline components/RecipeName.js]: Are we changing it from Title to Recipe Name?

File: styles/theme.style.js [hunk 3]
```diff
@@ -17,7 +17,11 @@ export default {
     WHITE_FONT_COLOR: "#FFFFFF",
     INPUT_BACKGROUND_COLOR: "#FFFFFF",
     INPUT_BORDER_COLOR: "#D4D4D4",
+    INPUT_BORDER_HIGHLIGHT_COLOR: "#3FA2F7",
+    INPUT_BORDER_WIDTH: 1,
     INPUT_BORDER_RADIUS: 6,
+    INPUT_HEIGHT: 36,
+    INPUT_WIDTH_SHORT: 48,
```
Comment (kmcknight1) [inline styles/theme.style.js]: NICE

File: components/RecipeName.js (hunk 2 above)
Comment (dlhauer) [reply components/RecipeName.js]: Ryan's design says "Recipe Name."

Comment (kmcknight1) [reply components/RecipeName.js]: My only concern is conflicting naming across our app. If we change it in one spot, we should change it in every instance to maintain consistency.

**Pull request merged**